<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_a">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_b">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_gr">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Градусы</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_r">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Радианы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_a">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_b">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y /_</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_c">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_res">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>74</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти  с</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
